src/Razor/src/Microsoft.VisualStudio.LanguageServerClient.Razor/DefaultRazorLanguageServerCustomMessageTarget.cs

```diff
@@ -229,6 +229,52 @@ public override async Task<RazorDocumentRangeFormattingResponse> RazorDocumentFo
             return response;
         }
 
+        public override async Task<RazorDocumentRangeFormattingResponse> RazorDocumentOnTypeFormattingAsync(DocumentOnTypeFormattingParams request, CancellationToken cancellationToken)
+        {
+            var response = new RazorDocumentRangeFormattingResponse() { Edits = Array.Empty<TextEdit>() };
+
+            await _joinableTaskFactory.SwitchToMainThreadAsync(cancellationToken);
```

Comment:
Hmm, I wonder if we actually need the main thread here. I think we might not

Reply:
I don't write code, I just copy it 😛

Reply:
😆 ya I think we can nuke this